extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/YoutubeParsingHelper.java

```diff
@@ -353,6 +353,11 @@ public static String getKey() throws IOException, ExtractionException {
         return key;
     }
 
+    static void resetClientVersionAndKey() {
```

Comment:
Maybe add a comment saying that this is only to be used for tests.

Reply:
I wouldn't add it, since android studio can already show you where a method is used. In this case it's only tests.
Additionally i did not make it public to only allow usage inside the extractor

Reply:
Ok, but I'd like to be able to tell just by looking at it that the method should not be used except for tests

Reply:
added